### ISSUE TITLE
Remove absolute slash prefix from bower.json

### ISSUE THREAD

**Issue by darklow** [epochjs/epoch#171]
Looks like absolute prefix slash is redundant in your `bower.json` config.
Therefore some other libraries fail to work nicely with epoch.

For example:
https://github.com/ck86/main-bower-files/issues/85#issuecomment-100137353

I checked other bower libs and almost no one uses `/` prefix.
Would you mind changing:
```
  "main": [
    "/epoch.min.js",
    "/epoch.min.css"
  ],
```
to
```
  "main": [
    "epoch.min.js",
    "epoch.min.css"
  ],
```

Thanks.

**Comment by tep-pl** [comment]
+1

**Comment by Twintails** [comment]
If you need the "/" in your final path string it should be trailing the predecessor. 
i.e.  "some/path/to/" + "file.tld"
or    "include/javascripts/" + "**/*.js

ref: font-awesome bower.json
https://github.com/FortAwesome/Font-Awesome/blob/master/bower.json


The bower.json is telling the machine to look at the root directory.
 "/" = machine root 
please consider "./" to reference parent directory to the file or remove the "/" before the file name. 

ex:
"main": [
    "./epoch.min.js",
    "./epoch.min.css"
  ],
  "license": "MIT",
  "ignore": [
    "**/.*",
    "./src",
    "./sass",
    "./tests",
    "./Cakefile",
    "./CHANGELOG.md",
    "./package.json"
  ]

**Comment by rsandor** [comment]
Fixed by #168
